elodie/filesystem.py

```diff
@@ -213,13 +218,38 @@ def process_file(self, _file, destination, media, **kwargs):
             shutil.move(_file, dest_path)
             os.utime(dest_path, (stat.st_atime, stat.st_mtime))
         else:
-            shutil.copy2(_file, dest_path)
+            shutil.copy(_file, dest_path)
+            self.set_date_from_filename(dest_path)
 
         db.add_hash(checksum, dest_path)
         db.update_hash_db()
 
         return dest_path
 
+    def set_date_from_filename(self, file):
```

Comment:
i'd love to do this, but I am not perl crack :) I only used copy and paste, the function was already existing in the same file as set_date_from_path_video. I could not figure out how to make this flexible to be used for video and pictures :) Happy to do it with some help or you can take this forward.